src/test/java/com/microsoft/azure/spring/data/cosmosdb/repository/integration/RoleRepositoryCollectionIT.java

```diff
@@ -63,43 +88,87 @@ public void testFindAllById() {
     }
 
     @Test
-    public void testFindById() {
-        this.repository.findById(DOMAIN.getNumber());
+    public void testCount() {
+        assert repository.count() == 0;
+        this.repository.save(DOMAIN);
+        assert repository.count() == 1;
     }
 
     @Test
-    public void testCount() {
-        this.repository.count();
+    public void testDeleteById() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/Microsoft/spring-data-cosmosdb/pullRequest?prid=4925697)